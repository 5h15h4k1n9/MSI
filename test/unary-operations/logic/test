logicalNot(false)
logicalNot(true)
exit
