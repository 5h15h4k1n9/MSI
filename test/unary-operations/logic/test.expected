(bool) true
(bool) false
Bye-bye!
